lumigator/schemas/lumigator_schemas/jobs.py

```diff
@@ -66,6 +74,170 @@ class JobSubmissionResponse(BaseModel):
     driver_node_id: str | None = None
     driver_exit_code: int | None = None
 
+    # Key used to identify the entrypoint in data sent to Ray.
```

Comment:
This should be placed imho somewhere else. The schemas are meant to be declarative schemas, and hold as little impl as possible.